docs/reference/commandline/cli.md

```diff
@@ -214,6 +214,7 @@ be set for each environment:
 | `httpsProxy`   | Default value of `HTTPS_PROXY` and `https_proxy` for containers, and as `--build-arg` on `docker build` |
 | `ftpProxy`     | Default value of `FTP_PROXY` and `ftp_proxy` for containers, and as `--build-arg` on `docker build`     |
 | `noProxy`      | Default value of `NO_PROXY` and `no_proxy` for containers, and as `--build-arg` on `docker build`       |
+| `allProxy`     | Default value of `ALL_PROXY` and `all_proxy` for containers, and as `--build-arg` on `docker build`     |
```

Comment:
Question: should we add ALL_PROXY to the environment variables section too? https://github.com/docker/cli/pull/3074/files#diff-98c80ce7ccfb31b6e6204cc1abaa7d250c8d92ba9f882a0f2b38115081c646d1L87

Reply:
Hm.. not sure; that section talks about what Go itself does by default (https://pkg.go.dev/net/http#ProxyFromEnvironment), which doesn't include `ALL_PROXY` (although I _think_ we have some code in place to support it in github.com/docker/go-connections).

It also mentions "_In particular_, you may find these useful:", to highlight the most common ones.

We can have a look though if `ALL_PROXY` is indeed supported (and if so, update that section), but it's separate from this one (which is for containers, not the daemon and cli itself)

Reply:
Ok let's keep it that way then 👍